docs/syntax.md

```diff
@@ -581,7 +581,7 @@ i24 = -42
 u24 = 42
 ```
 
-Currently, the 3 number types cannot be mixed.
+Currently, the 3 number types cannot be mixed, but can be converted in order to correspond to each other.
```

Comment:
We can't write operations that mix two types of number but we can explicitly convert between them